scripts/qa_crowbarsetup.sh

```diff
@@ -1662,14 +1662,7 @@ EOF
         service smb restart
     fi
 
-    sleep 20
-    if ! curl -m 59 -s $crowbar_api  > /dev/null || \
-        ! curl -m 59 -s --digest --user crowbar:crowbar $crowbar_api  | \
```

Comment:
as the function onadmin_is_crowbar_api_available is already in use (and works) this rather means that this function is missing the digest case (and it was not needed for the current usages of this function), right?